+neuron/Object.m

```diff
@@ -141,12 +141,6 @@ function list_methods(self)
             % Are we trying to directly access a class property?
             elseif (isa(method, "char") && length(S) == 1 && isprop(self, method))
                 [varargout{1:nargout}] = self.(method);
-            % Special case: size
-            % If we make an array of Objects, and ask for its size, Matlab
-            % throws an error if we don't exclude this special case here.
-            % TODO: check behavior for Objects other than Vector.
-            elseif (method == "size")
-                % Do nothing; method is replaced by self.length().
```

Comment:
Besides `Vector`, there is also a `.size` method for: `PlotShape`, `Shape`, `PtrVector`, `VBox`, `HBox`, and `Graph`.